lib/rouge/lexers/irb.rb

```diff
@@ -23,7 +23,7 @@ def lang_lexer
       end
 
       def prompt_regex
-        /^.*?(irb|pry).*?[>"*]/
+        /^.*?(irb|pry|>).*?[>"*]/
```

Comment:
**suggestion:** This will incorrectly pick up a section of an output block as the prompt.

![Screenshot 2023-06-04 at 10 39 29 am](https://github.com/rouge-ruby/rouge/assets/756722/6d5523fd-278c-429c-af42-c1bd0a3fa25a)

I also suggest adding support for multi-line for simple prompts.

```rb
"> puts <<XYZ
"> a
"> b
>> XYZ
a
b
=> nil
```

What do you think about the following adjustment?

```suggestion
        %r(
          ^.*?
          (
            (irb|pry).*?[>"*] |
            [>"*]>
          )
        )x
```

Reply:
Thanks for this fix and improvement! I wasn't even aware of multi-line prompts, to be honest… This is much better as my crude initial go at it. 🙏